### DISCUSSION TITLE
is there a way to move the table of contents to the right-hand side of the screen?

### DISCUSSION THREAD

**Discussion by ghost** [outline/outline#6035] [General] [answered]
See screenshot. I don't see any options that allow me to change the position of this table of contents.
![image](https://github.com/outline/outline/assets/77940332/312f0f9d-e74f-4050-9f37-1947b41b59ef)

**Comment by tommoor** [comment]
Not currently

**Reply by ghost** [reply]
I've added a FR.
https://github.com/outline/outline/discussions/6037

**Comment by ReessKennedy** [comment]
+1 ... I think this is actually a kind of important option. Consider that in-page table of contents are very often on the right side. Clickup's is on the right side, Dokuwiki, HackMD and WikiJS has an option that allows one to switch between left or right table of content positions. 

Great work, as always, @tommoor tommoor.

**Comment by EduNetEurope** [comment]
Duplicate of https://github.com/outline/outline/discussions/3214

**Comment by ReessKennedy** [comment]
There's actually a simple way to do this. ✅ 🎉 ... not sure why I didn't figure this out sooner. Wrote up solution at link below but's basically just a CSS mod with optional JS addition to make the table of contents show up by default: 

[https://www.reesskennedy.com/right-sidebar-outline-wiki/](https://www.reesskennedy.com/right-sidebar-outline-wiki/)

![BeforeAndAft](https://github.com/outline/outline/assets/3970267/4a92b4bc-0db5-4d2b-a3c4-b52ebbc3aaa3)

**Comment by EduNetEurope** [comment]
@tommoor 
I think you can close this request. :-)

**Comment by tommoor** [comment, marked as answer]
This feature was added as a workspace preference in https://github.com/outline/outline/releases/tag/v0.78.0
